posthog/queries/funnels/funnel_event_query.py

```diff
@@ -112,17 +86,35 @@ def get_query(self, entities=None, entity_name="events", skip_entity_filter=Fals
 
         null_person_filter = f"AND notEmpty({self.EVENT_TABLE_ALIAS}.person_id)" if self._using_person_on_events else ""
 
-        query = f"""
-            SELECT {', '.join(_fields)} FROM events {self.EVENT_TABLE_ALIAS}
+        select_fragment = (
+            f"SELECT {', '.join(_fields)}" + "{extra_select_fields}" + f" FROM events {self.EVENT_TABLE_ALIAS}"
```

Comment:
You can do the following in f-strings: `{{extra_select_fields}}` if really needed, though the logic behind it is sus.

Reply:
updated to this